browser.js

```diff
@@ -0,0 +1,20 @@
+/** @module  audio-speaker/browser */
+'use strict'
+
+var objectAssign = require('object-assign')
+var createContext = require('audio-context')
+var createWriter = require('web-audio-write')
+
+module.exports = function createSpeaker (opts) {
+  var options = {}
+
+  options = objectAssign({
+    channels: 1,
```

Comment:
I think this would go in line with [your recent audio-buffer changes](https://github.com/audiojs/audio-buffer/pull/16), right?  I'm for keeping this.  :+1: